### ISSUE TITLE
Library search bar is unused and instead always opens modal with another bar

### ISSUE THREAD

**Issue by DoumanAsh** [cppalliance/temp-site#314]
Subj.
I'm talking about search on this page https://i.imgur.com/1rlhTQ7.png

Right now it is not used (clicking on it will bring modal with search bar), if you enter anything in modal search bar, it is not reflected in page's search bar too.
So basically it is useless

On side note, I think modal is not needed, and search bar should be used instead as it is, without extra modals like this:
https://i.imgur.com/XHNdneE.png

**Comment by gregnewman** [comment]
There's another issue to implement a keyboard shortcut to trigger the command palette (modal) for search.

**Comment by DoumanAsh** [comment]
Oh I understand, well, if it is not intended and click action will be removed from page's search bar, then you can close I guess.
Just don't forget to do it once modal search bar is ready
